README.md

```diff
@@ -184,6 +184,83 @@ $ ./bin/publish.sh
 
 Please note that this will require all changes to be committed before using it.
 
+## Data Playground
+
+To add a new data source to the Data Playground, use the hugo new command as follows:
+
+```
+$ hugo new data-playground/$data-source-name.md
+```
+
+This will create a new file in the `content/en/data-playground/` directory that you may open and edit. When creating a new data source,, Hugo uses the `archetypes/data-playground.md` template file and generates the following frontmatter:
+
+```
+---
+title: "New Data Source"
+slug: "new-data-source"
+subtitle: "Add Data Type Here" 
+draft: true
+image: img/data-playground/data-source-img.png
+summary: "Add Summary To Display On List Page Here"
+github_link: "https://github.com/rotationalio/[insert-link-here]"
+description: "Add About This Data Source Description Here"
+producer_name: "Add Producer Name Here"
+producer_link: "https://producer-website.com"
+data: "Add Details About the Data Provided Here"
+is_account_required: false
+license: "Add Data Source License Type Here (Ex. Free, Commercial)"
+license_warning: Please review terms and conditions. Access to data sources can change.
+is_api_key_required: false
+api_type: "Add API Type(s) Provided Here and Separate Multiple Types With A Comma"
+sdks: "Add Available SDKs Here and Separate Multiple SDKs With A Comma"
+limits: "Add API Limit Information Here, If Available"
+
+---
+```
+
+Next, update the following fields:
+`subtitle`,
+`image`,
+`summary`,
+`github_link`,
+`description`,
+`producer_name`,
+`producer_link`,
+`data`,
+`is_account_required`,
+`license`,
+`is_api_key_required`,
+`api_type`,
+`sdks`,
+`limits`
+
+After updating the fields listed above, content added for the new data source should be added to the `static/index.json` file to update the search index.
+To update the `static/index.json` file, use the following command:
+
+```
+$ yarn index
+```
+
+After running the command, you should see that the `static/index.json` file has been updated and should include the following for the new data source:
+```
+ {
+        "uri": "/data-source-slug",
+        "content": "Data source content",
+        "tags": []
+  },
+```
+
+The `static/index.json` file will include content from the `/_index.md` file. This should be removed from the `static/index.json` file so that users will not see results from the page as it does not contain searchable content.
```

Comment:
I'm not sure if I understand what exactly needs to be removed. If I'm following, when `yarn index` runs, it is going to 
1. create a new `/_index.md` somewhere in the repo
2. add unsearchable content from `/_index.md` into `static/index.md`. 

Is that right? Can you clarify or provide an example of what unsearchable content may be automatically added from `/_index.md` into `static/index.json`? It might also help to show what the `static/index.json` should look like when the unsearchable content has been removed.

Reply:
Running `yarn index` will not create a new `/_index.md`. However, any frontmatter in that file will be included in the `static/index.json` file since the `/_index.md` file is in the `content/en/data-playground` directory. 

The `/_index.md` file is the `/data-playground` page. If a user searches were to search for `data` in the search bar, they'd see a result for the `/data-playground` page in the search results if it's not removed from `static/index.json` 

Lines 255 - 261 display how the `/_index.md` file appears in the `static/index.json/` file. However, I did remove the content that would appear (Ex. the link and page title) and replaced it with `index content`.